definitions/contested/json/CaseEventToFields/CaseEventToFields.json

```diff
@@ -4186,7 +4188,7 @@
     "PageDisplayOrder": 2,
     "PageColumnNumber": 1,
     "ShowSummaryChangeOption": "Y",
-    "CallBackURLMidEvent": "${CCD_DEF_COS_URL}/case-orchestration/ccdMidEvent",
+    "CallBackURLMidEvent": "${CCD_DEF_COS_URL}/case-orchestration/amend-application-app",
```

Comment:
What about the code that was previously executed by the mid-event handler? AmendApplicationContestedMidHandler

Reply:
Good spot, I've moved the postalService validation into the controller here [2e125d6](https://github.com/hmcts/finrem-case-orchestration-service/pull/1775/commits/2e125d6fc3a1b475a0994d7becb9b0d8e169b78c)